tasks/integrated-benchmark/src/work_env.rs

```diff
@@ -45,16 +50,16 @@ impl WorkEnv {
         &self.repository
     }
 
-    fn revision_root(&self, revision: &str) -> PathBuf {
-        self.root().join(revision)
+    fn sub_dir_path(&self, sub_dir: SubDir) -> PathBuf {
+        self.root().join(sub_dir.to_string())
     }
 
-    fn revision_install_script(&self, revision: &str) -> PathBuf {
-        self.revision_root(revision).join("install.bash")
+    fn sub_install_script(&self, sub_dir: SubDir) -> PathBuf {
```

Comment:
what does sub_install_script mean? does it mean `install_script_in_sub_dir`?

Reply:
Actually, I think just `script` will be a better name.

Reply:
> `install_script_in_sub_dir`

This name makes it look like it's installing some script in some directory, while in fact, it's just getting the path to an `install.bash` script.

Reply:
It has been renamed to `script_path`.